### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by sedrubal**
Ich konnte es nicht lassen ;)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# container ids
+*.running_container_id*
+
+# Created by https://www.gitignore.io/api/python,pycharm
```
Comment (mgmax) [inline .gitignore]: license?

Comment (sedrubal) [inline .gitignore]: I think MIT as the content of the generated gitignores comes from [github/gitignore](https://github.com/github/gitignore).

Gitignore.io itself uses GPL I think: https://github.com/joeblau/gitignore.io#license

Comment (mgmax) [inline .gitignore]: well, then please add the license information to the .gitignore and include a copy of https://github.com/github/gitignore/blob/master/LICENSE somewhere (maybe renamed to LICENSE.gitignore or so).

File: example-config/kastenwesen_config.py [hunk 2]
```diff
@@ -1,12 +1,42 @@
...
+    links=[test1],
+    docker_options="-p 1232:1234",
+    tests={'sleep_before': 0, 'ports': [1232]})
 config_containers.append(test2)
-portforwarder_to_test2 = DockerContainer("portforwarder-to-test2", "./portforwarder-to-test2/", links=[test2], docker_options="-p 1337:1234", tests={'sleep_before':0, 'ports': [1337]})
-config_containers.append(portforwarder_to_test2)
\ No newline at end of file
+
+##########################
+# portforwarder_to_test2 #
```
Comment (mgmax) [inline example-config/kastenwesen_config.py]: I don't really get the point of these fancy comments, they are only repeating what you can read in the source. Some _real_ explanation what is going on would be more helpful.

**Comment by mgmax** [comment]
rebased+merged, bis auf die .gitignore

**Comment by mgmax** [comment]
obacht, rebase und force-push hier

File: .gitignore (hunk 1 above)
Comment (sedrubal) [inline .gitignore]: Ok I see, then we should not use this generated gitignore as it is not reasonable to add such multiple files for only a gitignore.

**Comment by sedrubal** [comment]
obacht, noch ein force push :D dafür jetzt ein sinnvolles .gitignore...

**Comment by sedrubal** [comment]
f28937b kann man nehmen, wenn man mag, kann man aber auch weg schmeißen ;)

**Pull request merged**